app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -0,0 +1,238 @@
+              ]}
+              formRefs={formRefs}
+            />
+            <SavingIndicator
+              isSaved={
+                !isUpdatingEmissionsIntensityReportFormChange &&
+                !isAddingEmissionsIntensityReportFormChange
+              }
+            />
+          </header>
+          <FormBase
+            id="TEIMP_ReportingRequirementForm"
+            ref={(el) =>
+              (formRefs.current[reportingRequirementFormChange.id] = el)
+            }
+            validateOnMount={
+              reportingRequirementFormChange?.changeStatus === "staged"
+            }
+            idPrefix="TEIMP_ReportingRequirementForm"
+            schema={emissionIntensityReportingRequirementSchema as JSONSchema7}
+            formData={reportingRequirementFormChange?.newFormData}
+            formContext={{
+              form: reportingRequirementFormChange?.newFormData,
+            }}
+            uiSchema={emissionIntensityReportingRequirementUiSchema}
+            onChange={(change) =>
+              handleChange(
+                reportingRequirementFormChange,
+                change.formData,
+                "pending"
+              )
+            }
+          />
+          <FormBase
+            id="TEIMP_EmissionIntensityReportForm"
+            ref={(el) =>
+              (formRefs.current[emissionIntensityReportFormChange.id] = el)
+            }
+            validateOnMount={
+              emissionIntensityReportFormChange?.changeStatus === "staged"
+            }
+            idPrefix="TEIMP_EmissionIntensityReportForm"
+            schema={emissionIntensityReportSchema as JSONSchema7}
+            formData={emissionIntensityReportFormChange?.newFormData}
+            formContext={{
+              form: emissionIntensityReportFormChange?.newFormData,
+            }}
+            uiSchema={emissionIntensityReportUiSchema}
+            onChange={(change) =>
+              handleChange(
+                emissionIntensityReportFormChange,
+                change.formData,
+                "pending"
+              )
+            }
+          />
+          <Button
+            size="medium"
+            variant="primary"
+            onClick={() =>
+              stageReportFormChanges(
+                updateEmissionsIntensityReportFormChange,
+                props.onSubmit,
+                formRefs,
+                [
+                  ...projectRevision
+                    .emissionIntensityReportingRequirementFormChange.edges,
+                  ...projectRevision.emissionIntensityReportFormChange.edges,
+                ],
+                "TEIMP",
+                updateFormChange
+              )
+            }
+            disabled={
+              isAddingEmissionsIntensityReportFormChange ||
+              isUpdatingEmissionsIntensityReportFormChange ||
+              isUpdatingFormChange
+            }
+          >
+            Submit Milestone Reports
```

Comment:
`Submit Milestone Reports` isn't right, but not sure what the button text should be. @suhafa or @nanyangpro?